src/System Application/App/Guided Experience/src/Guided Experience/GuidedExperienceImpl.Codeunit.al

```diff
@@ -521,6 +521,39 @@ codeunit 1991 "Guided Experience Impl."
         end;
     end;
 
+    procedure GetContentForManualSetup(var GuidedExperienceItemTemp: Record "Guided Experience Item" temporary)
```

Comment:
```suggestion
    procedure GetContentForManualSetup(var TempGuidedExperienceItem: Record "Guided Experience Item" temporary)
```

Reply:
In order to keep consistency, should it also be changed at procedures above and below?
procedures: GetContentForSetupPage and GetContentForAssistedSetup

Reply:
That would be nice, yes.

Reply:
(not a must though for your PR)

Reply:
Nah, already done, no problem. But seems like I will have to adjust system test app due removal of fields :)